### DISCUSSION TITLE
What is JSX?

### DISCUSSION THREAD

**Discussion by smartman0307** [community/community#57129] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

What is JSX in JavaScript?

**Comment by SuperDev314** [comment]
JSX is a XML-like syntax extension to ECMAScript (the acronym stands for JavaScript XML). Basically it just provides syntactic sugar for the React.createElement() function, giving us expressiveness of JavaScript along with HTML like template syntax.

In the example below text inside <h1> tag is returned as JavaScript function to the render function.
class App extends React.Component { 
	render() { 
		return( 
			<div> 
				<h1>{'Welcome to React world!'}</h1> 
			</div> 
		) 
	} 
}

**Comment by smartman0307** [comment]
Thanks very much! You are right.

**Comment by tuves** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
